pkg/network/encoding/http.go

```diff
@@ -20,6 +21,9 @@ type httpEncoder struct {
 	staticTags     map[types.ConnectionKey]uint64
 	dynamicTagsSet map[types.ConnectionKey]map[string]struct{}
 
+	dataPool  []model.HTTPStats_Data
```

Comment:
Did you look at [sync.Pool](https://pkg.go.dev/sync#Pool) for this?

Reply:
done